application/performance/locust/files/ccs_50_destinations.json

```diff
@@ -0,0 +1 @@
, {"reference": "destination13", "latitude": 51.368998, "longitude": -0.805694}, {"reference": "destination14", "latitude": 52.035898, "longitude": -0.79788}, {"reference": "destination15", "latitude": 51.764579, "longitude": -0.743976}, {"reference": "destination16", "latitude": 51.528911, "longitude": -0.751278}, {"reference": "destination17", "latitude": 52.056265, "longitude": -0.853194}, {"reference": "destination18", "latitude": 51.731757, "longitude": -1.217622}, {"reference": "destination19", "latitude": 52.170476, "longitude": -0.505326}, {"reference": "destination20", "latitude": 51.588423, "longitude": -1.425644}, {"reference": "destination21", "latitude": 51.580289, "longitude": -0.549317}, {"reference": "destination22", "latitude": 52.030166, "longitude": -2.104857}, {"reference": "destination23", "latitude": 51.762444, "longitude": -1.194063}, {"reference": "destination24", "latitude": 51.708689, "longitude": -1.781017}, {"reference": "destination25", "latitude": 51.754816, "longitude": -1.261454}, {"reference": "destination26", "latitude": 52.059401, "longitude": -1.340952}, {"reference": "destination27", "latitude": 51.992499, "longitude": -2.157838}, {"reference": "destination28", "latitude": 51.605067, "longitude": -1.221519}, {"reference": "destination29", "latitude": 51.898941, "longitude": -2.074691}, {"reference": "destination30", "latitude": 51.773188, "longitude": -0.457292}, {"reference": "destination31", "latitude": 52.048836, "longitude": -1.328231}, {"reference": "destination32", "latitude": 51.997414, "longitude": -0.747638}, {"reference": "destination33", "latitude": 51.746945, "longitude": -0.97809}, {"reference": "destination34", "latitude": 51.222445, "longitude": -1.529172}, {"reference": "destination35", "latitude": 51.348496, "longitude": -1.669545}, {"reference": "destination36", "latitude": 51.95269, "longitude": -1.965279}, {"reference": "destination37", "latitude": 52.026193, "longitude": -0.493277}, {"reference": "destination38", "latitude": 51.814526, "longitude": -0.812337}, {"reference": "destination39", "latitude": 51.141136, "longitude": -0.490033}, {"reference": "destination40", "latitude": 51.688163, "longitude": -1.275136}, {"reference": "destination41", "latitude": 51.426939, "longitude": -0.677163}, {"reference": "destination42", "latitude": 51.996295, "longitude": -0.722886}, {"reference": "destination43", "latitude": 52.064688, "longitude": -1.343107}, {"reference": "destination44", "latitude": 51.597361, "longitude": -1.805169}, {"reference": "destination45", "latitude": 51.645972, "longitude": -0.779125}, {"reference": "destination46", "latitude": 51.898957, "longitude": -2.080047}, {"reference": "destination47", "latitude": 51.173477, "longitude": -1.779751}, {"reference": "destination48", "latitude": 51.531648, "longitude": -0.582928}, {"reference": "destination49", "latitude": 51.66126, "longitude": -0.751114}, {"reference": "destination50", "latitude": 51.528097, "longitude": -0.724026}]}
```

Comment:
This comment is for all the ccs_xx_destination.json files in locust/file/
The config now points to the files held in ccs_prefix = "../../mock/requests/" so are these files required any more?